modules/Bio/EnsEMBL/Variation/DBSQL/vcf_config.json

```diff
@@ -27,12 +27,12 @@
             "sample_prefix": "1000GENOMES:phase_3:"
         },
         {
-            "id": "gnomADg_GRCh37",
-            "description": "Genome Aggregation Database genomes r2.1",
+            "id": "gnomADg_v4.1_GRCh37",
+            "description": "Genome Aggregation Database genomes v4.1",
             "species": "homo_sapiens",
             "assembly": "GRCh37",
             "type": "remote",
-            "filename_template": "https://ftp.ensembl.org/pub/data_files/homo_sapiens/GRCh37/variation_genotype/gnomad/r2.1/genomes/gnomad.genomes.r2.1.sites.chr###CHR###_noVEP.vcf.gz",
+            "filename_template": "https://ftp.ensembl.org/pub/data_files/homo_sapiens/GRCh37/variation_genotype/gnomad/v4.1/genomes/gnomad.genomes.v4.1.sites.grch37.chr###CHR###_trimmed_liftover.vcf.gz",
```

Comment:
Couple of points on this:
* There is no `v4.1` folder in the public FTP inside GRCh37's gnomad folder: https://ftp.ensembl.org/pub/data_files/homo_sapiens/GRCh37/variation_genotype/gnomad/
* In the files inside the `ensemblftp` directory, the name of the files currently ends in `chrX_liftover.vcf.gz`, there is no `_trimmed`.

Reply:
Thanks for taking care of this, @nakib103!